### PR TITLE
Kustomize bulk-scan-payment-processor

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[Deduplication in bulk scan payment processor cnp-flux-config](https://tools.hmcts.net/jira/browse/BPS-1355)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/bsp/kustomization.yaml [hunk 1]
```diff
@@ -2,5 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: bsp
 bases:
-- namespace.yaml
-# Warning : Adding a file here, adds to all environments to which you add your kustomization.
+  - namespace.yaml
+  # Warning : Adding a file here, adds to all environments to which you add your kustomization.
+  - bulk-scan-payment-processor/bulk-scan-payment-processor.yaml
```
Comment (adusumillipraveen) [inline k8s/namespaces/bsp/kustomization.yaml:7]:
```suggestion
  - bulk-scan-payment-processor/bulk-scan-payment-processor.yaml
  # Warning : Adding a file here, adds to all environments to which you add your kustomization.
```

File: k8s/aat/common-overlay/bsp/kustomization.yaml [hunk 2]
```diff
@@ -2,4 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: bsp
 bases:
-- ../../../namespaces/bsp
+  - ../../../namespaces/bsp
+patchesStrategicMerge:
+  - ../../../namespaces/bsp/bulk-scan-payment-processor/aat.yaml
```
Comment (adusumillipraveen) [inline k8s/aat/common-overlay/bsp/kustomization.yaml:7]: couldn't see this file

Comment (doncem) [reply k8s/aat/common-overlay/bsp/kustomization.yaml:7]: good catch. i've used as `git mv` on most files. this one probably was manually created and i missed it ;)

File: k8s/namespaces/bsp/bulk-scan-payment-processor/aat.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+kind: HelmRelease
+metadata:
+  name: bulk-scan-payment-processor
+  namespace: bsp
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:prod-*
+    repository.fluxcd.io/smoke: java.smoketests.image
+    filter.fluxcd.io/smoke: glob:prod-*
```
Comment (adusumillipraveen) [inline k8s/namespaces/bsp/bulk-scan-payment-processor/aat.yaml]: Can u refer to draft store for these, it changed slightly.

File: k8s/aat/common-overlay/bsp/kustomization.yaml (hunk 2 above)
Comment (adusumillipraveen) [reply k8s/aat/common-overlay/bsp/kustomization.yaml:7]: Anyway, PR checks failed :)

File: CODEOWNERS [hunk 4]
```diff
@@ -238,10 +238,9 @@ k8s/**/common/ethos/identity.yaml @hmcts/ethos
 k8s/**/common/ethos/repl-docmosis-service.yaml @hmcts/ethos
 k8s/aat/common/ethos/ecm-consumer.yaml @hmcts/ethos
 
-### Bulk scan and print
+### Bulk scan
 k8s/namespaces/bsp/ @hmcts/bulk-scan
 k8s/**/common/bsp/identity.yaml @hmcts/bulk-scan
-k8s/**/common/bsp/bulk-scan-payment-processor.yaml @hmcts/bulk-scan
```
Comment (nitinprabhu) [inline CODEOWNERS:244]: any reason this is removed from codeowners ?

Comment (doncem) [reply CODEOWNERS:244]: everything has moved to `namespace/bsp` now

**Comment by doncem** [comment]
not sure what's so scary. kustomize already aplied to few services and works they way before with less duplication in place. here's a generated yaml using `kustomize` command line for prod:

```yaml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    hmcts.github.com/envInjector: enabled
    slackChannel: bsp-build-notices
  name: bsp
---
apiVersion: helm.fluxcd.io/v1
kind: HelmRelease
metadata:
  annotations:
    filter.fluxcd.io/java: glob:prod-*
    filter.fluxcd.io/java.smoketests: glob:prod-*
    fluxcd.io/automated: "true"
    repository.fluxcd.io/java: java.image
    repository.fluxcd.io/java.smoketests: java.smoketests.image
  name: bulk-scan-payment-processor
  namespace: bsp
spec:
  chart:
    name: bulk-scan-payment-processor
    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
    version: 0.2.2
  releaseName: bulk-scan-payment-processor
  values:
    java:
      environment:
        IDAM_API_URL: https://idam-api.platform.hmcts.net
      functionaltests:
        image: hmctspublic.azurecr.io/bulk-scan/payment-processor-test:prod-dc64623f
      image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-dc64623f
      replicas: 2
      smoketests:
        enabled: true
        image: hmctspublic.azurecr.io/bulk-scan/payment-processor-test:prod-dc64623f
      testsConfig:
        environment:
          SLACK_CHANNEL: bsp-test-notices
          SLACK_NOTIFY_SUCCESS: false
          TEST_URL: http://bulk-scan-payment-processor-prod.service.core-compute-prod.internal
        memoryLimits: 3072Mi
        serviceAccountName: tests-service-account
      useInterpodAntiAffinity: true
```

File: CODEOWNERS (hunk 4 above)
Comment (adusumillipraveen) [reply CODEOWNERS:244]: It doesnt exist anymore. Also, u have full control on what files you add/edit in `k8s/namespaces/bsp/` , but u need approval when u add them to kustomizations.

**Pull request merged**